wgbs-pipeline.wdl

```diff
@@ -362,7 +368,13 @@ task map {
         mkdir -p fastq/${sample_name}
         cat ${write_lines(fastqs)} | xargs -I % ln % fastq/${sample_name}
         mkdir -p mapping/${sample_barcode}
-        gemBS -j ${gemBS_json} map -b ${sample_barcode} --ignore-db
+        gemBS \
+            -j ${gemBS_json} \
+            map \
+            -b ${sample_barcode} \
+            ${if defined(sort_threads) then ("--sort-threads " + sort_threads) else ""} \
+            ${if defined(sort_memory) then ("--sort-memory " + sort_memory) else ""} \
```

Comment:
There is a simpler expression for this. It's ignored if any variable in the expression is not defined.
```
            ${"--sort-threads " + sort_threads} \
            ${"--sort-memory " + sort_memory} \
```

Reply:
Nice, didn't know about that. To me it does look a little weird though, since I don't think concatenating `String` with `String?` is a valid WDL expression. Is it part of the spec? To me it seems like what you wrote should be an error.

I think I'll leave this as is since it is clearer IMO.